README.rst

```diff
@@ -512,6 +512,18 @@ setting ``DEFAULT_PRIORITY``. Integration with asynchronous email backends
         'DEFAULT_PRIORITY': 'now'
     }
 
+Override recipient
```

Comment:
`Override Recipients`

Reply:
I don't understand this review

Reply:
It's ok, I'll merge this PR in and touch up the docs later.